lib/publiccloud/provider.pm

```diff
@@ -352,6 +352,11 @@ sub create_instances {
             # Install server's ssh publicckeys to prevent authenticity interactions
             assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
         }
+
+        $instance->ssh_assert_script_run('sudo ip -6 a a ' . $instance->public_ip6 . '/128 dev eth0') if ($instance->public_ip6);
```

Comment:
Shouldn't this be excluded for Azure? Because `podman_ipv6` is excluded for Azure I assume that IPv6 is not available there

Reply:
no need for additional  `is_azure` condition here . Call will be excluded just because `$instance->public_ip6`  is undefined for Azure

Reply:
This has been discarded.